src/chainparams.cpp

```diff
@@ -100,8 +100,8 @@ class CMainParams : public CChainParams {
         pchMessageStart[3] = 0xd9;
         nDefaultPort = 8333;
         nPruneAfterHeight = 100000;
-        m_assumed_blockchain_size = 280;
-        m_assumed_chain_state_size = 4;
+        m_assumed_blockchain_size = 320;
+        m_assumed_chain_state_size = 6;
```

Comment:
My chainstate/ dir is 3.8 GB. Are you increasing this to give some room for expansion, or seeing different numbers than I do?

Reply:
How do you check the size of the chainstate dir?

```
$ du -sh ~/Downloads/bm/chainstate/
4.6G	/home/marco/Downloads/bm/chainstate/

Reply:
```
du -sh ~/.bitcoin/chainstate/
3.8G	/home/instagibbs/.bitcoin/chainstate/
```